src/components/flow/Flow.tsx

```
@@ -146,6 +146,11 @@ export class Flow extends React.PureComponent<FlowStoreProps, {}> {
     return (this.ghost = ref);
   }
 
+  public isMobile() {
+    const win = window as any;
+    return win.isMobile && win.isMobile();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)